calculator.go

```diff
@@ -68,6 +73,8 @@ func SizeInBytes(av *types.AttributeValue) (int, error) {
 	}
 }
 
+// Is size enough as name?
+// I think name could be shorter if you add some a comment with some docs.
```

Comment:
I think Size would be enough if there was a "size type" type in Go like `time.Duration`. Now there isn't, so I would prefer to state the unit the function like `SizeInBytes` rather than name it `Size` and rely on comment/docs.